code/modules/admin/terrainify.dm

```diff
@@ -689,6 +706,18 @@ ABSTRACT_TYPE(/datum/terrainify)
 			station_repair.clean_up_station_level(params["vehicle"] & TERRAINIFY_VEHICLE_CARS, params["vehicle"] & TERRAINIFY_VEHICLE_FABS)
 			handle_mining(params, space)
 
+			if(params["Spooky"])
+				var/list/station_areas = get_accessible_station_areas()
+				for(var/area_name in station_areas)
+					var/area/A = station_areas[area_name]
+					A.occlude_foreground_parallax_layers = TRUE
+					for(var/turf/T in get_area_turfs(A))
+						if(T.z == Z_LEVEL_STATION)
+							T.update_parallax_occlusion_overlay()
+							LAGCHECK(LAG_MED)
```

Comment:
Should the [caustics parallax PR](https://github.com/goonstation/goonstation/pull/16477) be merged, this block of code will be unnecessary, as within that PR, station areas will by default occlude foreground parallax layers.

Reply:
Dear Me,
Do this conditionally and then remove iff aforementioned issue is merged
Love,
Me